bucket/rainmeter-portable.json

```
@@ -0,0 +1,55 @@
+{
+    "homepage": "https://rainmeter.net/",
+    "description": "Rainmeter is a desktop customization tool for Windows.",
+    "version": "4.3.0.3298",
+    "license": "GPL-2.0-only",
+    "url": "https://github.com/rainmeter/rainmeter/releases/download/v4.3.0.3298/Rainmeter-4.3.exe#/inst.exe",
+    "hash": "1bb57129f6871c7e5a835ded406de7096aebbe53196006ca0d00073948fbb16b",
+    "depends": "7zip",
+    "architecture": {
+        "64bit": {
+            "installer": {
+                "script": "7z x \"$dir\\inst.exe\" -o\"$dir\" -aoa"
+            }
+        },
+        "32bit": {
+            "installer": {
+                "script": "7z x \"$dir\\inst.exe\" -o\"$dir\" -aos"
+            }
+        }
+    },
+    "pre_install": [
+        "if(!(Test-Path \"$persist_dir\\Rainmeter.ini\")) { New-Item \"$dir\\Rainmeter.ini\" -Type File | Out-Null }",
+        "if(!(Test-Path \"$persist_dir\\Rainmeter.data\")) { New-Item \"$dir\\Rainmeter.data\" -Type File | Out-Null }",
+        "if(!(Test-Path \"$persist_dir\\Rainmeter.stats\")) { New-Item \"$dir\\Rainmeter.stats\" -Type File | Out-Null }"
+    ],
+    "persist": [
+        "Rainmeter.ini",
+        "Rainmeter.data",
+        "Rainmeter.stats",
+        "Layouts",
+        "Skins",
+        "Plugins"
+    ],
+    "post_install": [
```

Tried to make it as readable as possible and added comments for other maintainers.
Array usage is better for these operations since you do not need to repeat $dir\....
```suggestion
    "post_install": [
        "    # Makes default welcome skins appear on new installation.",
        "if(!(Get-Content \"$dir\\Rainmeter.ini\")) {",
        "    @('Layouts\\illustro default\\Rainmeter.ini', 'Skins', 'Layouts') | ForEach-Object {",
        "        Copy-Item \"$dir\\Defaults\\$_\" \"$persist_dir\" -Force -Recurse | Out-Null",
        "    }",
        "}",
        "    # If there was update for plugin, copy default plugins after persisting.",
        "if (Test-Path \"$dir\\Plugins.original\") {",
        "    Copy-Item \"$dir\\Plugins.original\\*\" \"$persist_dir\\Plugins\" -Force -Recurse",
        "    Remove-Item \"$dir\\Plugins.original\" -Force -Recurse | Out-Null",
        "}",
        "@('$PLUGINSDIR', 'uninst.exe.nsis', 'inst.exe') | ForEach-Object { Remove-Item \"$dir\\$_\" -Force -Recurse }"
    ],
```